src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,27 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws RuntimeException {
+        if (user == null || user.getLogin() == null || user.getAge() == null
+                || user.getAge() < 0 || user.getPassword() == null) {
+            throw new RegistrationException("Invalid data!");
```

Comment:
```suggestion
    public User register(User user) {
        if (user == null || user.getLogin() == null || user.getAge() == null
                || user.getAge() < 0 || user.getPassword() == null) {
            throw new RegistrationException("Invalid data!");
```
Why are you throwing an exception?
I think that it is necessary to break the invalid possible values ​​into separate if conditions, in order to make the error message more informative, where exactly this error occurred